kahuna/public/js/components/gr-confirm-delete/gr-confirm-delete.js

```diff
@@ -6,10 +7,14 @@ confirmDelete.directive('grConfirmDelete', ['$timeout', function($timeout) {
 
     return {
         restrict: 'E',
+        transclude: true,
         template: `
-            <button class="button-ico" type="button" ng:click="showConfirm = true">
-                <span class="confirm" ng:if="showConfirm">Sure?</span>
-                <gr-icon>undo</gr-icon>
+            <button class="gr-confirm-delete button-ico" type="button"
+                ng:click="showConfirm = true"
+                ng:class="{'gr-confirm-delete--confirm': showConfirm}">
+                <gr-icon>delete</gr-icon>
+                <ng-transclude ng:if="!showConfirm"></ng-transclude>
+                <span class="confirm" ng:if="showConfirm">are you sure?</span>
```

Comment:
Just a note: let's keep separate functional changes in separate PRs, that makes it easier to review, revert, avoids blocking individual functionalities when discussing unrelated changes, etc. Thanks.